### ISSUE TITLE
Used transitions are seen as unused

### ISSUE THREAD

**Issue by ccampbell** [sveltejs/eslint-plugin-svelte3#5]
```html
<script>
import { fly } from 'svelte/transition';
const options = [10, 20, 30, 40, 50];
let number;
export let maxTabs;
</script>

{#if options.indexOf(maxTabs) === -1}
    <div in:fly="{{ y: -10, duration: 200 }}" out:fly="{{ y: -10, duration: 200 }}">
        <input bind:this="{number}" type="number" name="max" id="input-max" value="{maxTabs}">
    </div>
{/if}
```

```
 3:10  error  'fly' is defined but never used  no-unused-vars
```

**Comment by Conduitry** [comment]
It looks like this is the same as or a very similar issue to https://github.com/sveltejs/svelte/issues/2294

This plugin is relying on the Svelte compiler to report which variables are referenced in the template, and Svelte is reporting `false` in this case, which is incorrect.

**Comment by Conduitry** [comment]
This should be good to go in the next beta of Svelte.
